### PR TITLE
Add follow-up section and display control

### PR DESCRIPTION
**Pull request by nuhuhmutebi**
Added the section on the second registration screen and `follow up` display widget under clinical with the corresponding English and French translations.

### CODE REVIEW

File: configuration/openmrs/apps/clinical/dashboard.json [hunk 1]
```diff
@@ -559,5 +559,16 @@
         "numberOfVisits": 5
       }
     }
-  }
+  },
+  "followup": {
```
Comment (reagan-meant) [inline configuration/openmrs/apps/clinical/dashboard.json]: This should be a dashboard widget not a new dashboard

Comment (nuhuhmutebi) [reply configuration/openmrs/apps/clinical/dashboard.json]: Done

File: configuration/openmrs/apps/clinical/dashboard.json [hunk 2]
```diff
@@ -162,6 +162,17 @@
...
+        "translationKey": "DASHBOARD_TITLE_FOLLOW_UP_KEY",
+        "type": "observation",
+        "title": "Follow up",
+        "isObservation": true,
+          "displayOrder":1001,
+          "dashboardConfig": {
+            "conceptNames": ["Medical treatment started", "Comments", "Home assistance requested"],
+              "scope": "latest"
+          }
+      },
```
Comment (reagan-meant) [inline configuration/openmrs/apps/clinical/dashboard.json]: Model this to look like [this](https://github.com/mekomsolutions/bahmni-config-easydoc/blob/4fd1a1908b7ead337615c385e8aa59e118c2c3e3/configuration/openmrs/apps/clinical/dashboard.json#L289-L310)

**Review by reagan-meant — COMMENTED** [review]
Set the numberofVisits to 2.

File: configuration/openmrs/apps/clinical/dashboard.json (hunk 2 above)
Comment (nuhuhmutebi) [reply configuration/openmrs/apps/clinical/dashboard.json]: Done

Comment (reagan-meant) [reply configuration/openmrs/apps/clinical/dashboard.json]: @nuhuhmutebi  Could you set the visit number to 2

**Comment by nuhuhmutebi** [comment]
Great

**Pull request merged**